C-Plus-Plus/other/n_string_concatenate.cpp

```diff
@@ -0,0 +1,47 @@
+/*
+String Concatenation 
+Problem Statement : Take n string from user and concatenate them together.
+Input Format : First line takes number of strings to be concatenated.
+               Next n lines contains the strings to be concatenated.
+Output Format : Print a single line that contains the concatenated string.
+Note : No space is there in between a single string.
+
+Approach : Take n strings as input using loop.
+           Take a string variable and update it using append() function each time a new sting is entered.
+           Print the string.
+*/
+
+#include <bits/stdc++.h>
+using namespace std;
+
+int main() {
```

Comment:
Don't push the main logic of the code in the main() function. Use main() only for the driver code